src/cookies.rs

```diff
@@ -1,25 +1,109 @@
 //! Cookie-related functionality for WebDriver.
 use serde::{Deserialize, Serialize};
 use time::OffsetDateTime;
-use webdriver::command::WebDriverCommand;
+use webdriver::command::{AddCookieParameters, WebDriverCommand};
+use webdriver::common::Date;
 
 use crate::client::Client;
 use crate::error;
 
 /// Type alias for a [cookie::Cookie]
 pub type Cookie<'a> = cookie::Cookie<'a>;
 
+/// Wrapper for serializing AddCookieParameters.
+#[derive(Debug, Serialize)]
+pub struct AddCookieParametersWrapper<'a> {
+    /// The cookie to serialize.
+    #[serde(with = "AddCookieParameters")]
+    pub cookie: &'a AddCookieParameters,
+}
+
+/// The SameSite parameter for cookies.
+///
+/// See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite
+/// for more details.
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+pub(crate) enum SameSite {
```

Comment:
Can we not re-use `cookie::SameSite` here? Why do we need our own type when it's only `pub(crate)`?

Reply:
`cookie::SameSite` doesn't implement `Serialize` or `Deserialize`. But I can do this another way.